Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -196,91 +199,49 @@ theorem lift_pred (o : Ordinal.{v}) : lift.{u} (pred o) = pred (lift.{u} o) := b
 
 /-! ### Limit ordinals -/
 
-
 /-- A limit ordinal is an ordinal which is not zero and not a successor.
 
-TODO: deprecate this in favor of `Order.IsSuccLimit`. -/
+Deprecated: use `Order.IsSuccLimit` instead. -/
+@[deprecated IsSuccLimit (since := "2025-02-09")]
 def IsLimit (o : Ordinal) : Prop :=
   IsSuccLimit o
 
-theorem isLimit_iff {o} : IsLimit o ↔ o ≠ 0 ∧ IsSuccPrelimit o := by
-  simp [IsLimit, IsSuccLimit]
-
-theorem IsLimit.isSuccPrelimit {o} (h : IsLimit o) : IsSuccPrelimit o :=
-  IsSuccLimit.isSuccPrelimit h
-
-@[deprecated IsLimit.isSuccPrelimit (since := "2024-09-05")]
-alias IsLimit.isSuccLimit := IsLimit.isSuccPrelimit
-
-theorem IsLimit.succ_lt {o a : Ordinal} (h : IsLimit o) : a < o → succ a < o :=
-  IsSuccLimit.succ_lt h
-
-theorem isSuccPrelimit_zero : IsSuccPrelimit (0 : Ordinal) := isSuccPrelimit_bot
+theorem isSuccLimit_iff {o : Ordinal} : IsSuccLimit o ↔ o ≠ 0 ∧ IsSuccPrelimit o := by
+  simp [IsSuccLimit]
 
-@[deprecated isSuccPrelimit_zero (since := "2024-09-05")]
-alias isSuccLimit_zero := isSuccPrelimit_zero
+@[simp]
+theorem isSuccPrelimit_zero : IsSuccPrelimit (0 : Ordinal) :=
+  isSuccPrelimit_bot
```

Comment:
Can you revert changes like this? They make the diff much harder to read than necessary
```suggestion
theorem isSuccPrelimit_zero : IsSuccPrelimit (0 : Ordinal) := isSuccPrelimit_bot
```